xchange-gateio/src/main/java/org/knowm/xchange/gateio/dto/marketdata/GateioCandlestickHistory.java

```diff
@@ -0,0 +1,37 @@
+package org.knowm.xchange.gateio.dto.marketdata;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.knowm.xchange.gateio.dto.GateioBaseResponse;
+import java.util.List;
+
+public class GateioCandlestickHistory extends GateioBaseResponse {
+
+  private final List<List<String>> candlesticks;
+  private final String elapsed;
+
+  private GateioCandlestickHistory(
+      @JsonProperty("data") List<List<String>> candlesticks,
+      @JsonProperty("result") boolean result,
+      @JsonProperty("elapsed") String elapsed) {
+
+    super(result, null);
+    this.candlesticks = candlesticks;
+    this.elapsed = elapsed;
+  }
+
+  public List<List<String>>  getCandlesticks() {
+
```

Comment:
get rid of these extra spaces here and below

Reply:
Done